### ISSUE TITLE
You are not computing the loss on the test set

### ISSUE THREAD

**Issue by CAridorc** [ComputationalDomain/PINNs#2]
If I understood your code correctly, are not computing the loss on the test set, so you cannot know if you have overfitting to the training data.
